drivers/video/fb.c

```diff
@@ -1032,7 +1032,13 @@ static int fb_mmap(FAR struct file *filep, FAR struct mm_map_entry_s *map)
   if (map->offset >= 0 && map->offset < panelinfo.fblen &&
       map->length && map->offset + map->length <= panelinfo.fblen)
     {
+#ifdef CONFIG_BUILD_KERNEL
+      map->vaddr = vm_map_region((uintptr_t)panelinfo.fbmem,
+                                            panelinfo.fblen);
+      map->vaddr += map->offset;
```

Comment:
can vm_map_region support unaligned adress

Reply:
I don't know how to support unaligned address yet. It seems that at RISC-V MMU requires aligned addresses, also granule allocator operates on aligned address as well.

Reply:
it's simple to move the offset addition to vm_map_region

Reply:
@yf13 I mean passing (panelinfo.fbmem + map->offset) to vm_map_region.

Reply:
@yf13 this comment isn't addressed yet.

Reply:
Yes, just mask out offset before creating virtual-physical page setup, and add offset to the returned virtual address before returrn.

Reply:
Can you confirm which one below is what you mean:

a) new interface is ` vm_map_region(paddr, size, offset)`? here offset can be bigger than MM_PGSIZE. 
b) new interface is `vm_map_region(paddr+offset, size)`? here offset is smaller than MM_PGSIZE.
c) none-of above, please teach explicitly.

The offset in `man mmap()` is multiple of MM_PGSIZE or zero, is this the same in NuttX?

Reply:
nochangehere?

Reply:
rolled back to confirm  CI status...  just pushed an update supporting unaligned paddr mapping and moved `vm_map_xxx()` into ` vm_region.c`.